.github/workflows/reporting-e2e.yml

```diff
@@ -0,0 +1,49 @@
+name: Reporting End-to-End
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
Do we want this test to run every time or just when:
* reporting-server
* reporting
* react-component
* rmf-auth
* reporting-e2e 
packages are updated?

Reply:
i'm not sure what common convention is but it's alright to do it as you've suggested then I think that would be better!